requirements.txt

```diff
@@ -4,5 +4,8 @@ flake8
 # previews.
 rst_include
 
+# the PySpice module for rendering FET bins plots
+PySpice==1.4.3
```

Comment:
Does this version need to be hard coded?

Reply:
fixed